modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceListLocalServiceImpl.java

```diff
@@ -412,14 +412,10 @@ public void cleanPriceListCache(long companyId) {
 	@Indexable(type = IndexableType.DELETE)
 	@Override
 	@SystemEvent(type = SystemEventConstants.TYPE_DELETE)
-	public CommercePriceList deleteCommercePriceList(
-			CommercePriceList commercePriceList)
+	public CommercePriceList deleteForceCommercePriceList(
```

Comment:
We should call it this
```suggestion
	public CommercePriceList forceDeleteCommercePriceList(
```